### PR TITLE
add logos to docs

### PR DESCRIPTION
**Pull request by kecnry**
This PR adds logos created by @Jenneh to the docs and readme (and also adds a few missing badges to the readme).

[rendered docs](https://lcviz--48.org.readthedocs.build/en/48/)
[rendered readme](https://github.com/kecnry/lcviz/tree/docs-logo#readme)

### CODE REVIEW

File: docs/logos/lcviz.svg
Comment (bmorris3) [inline docs/logos/lcviz.svg:1]: YES! 🎉

**Comment by camipacifici** [comment]
it looks great!!

**Comment by bmorris3** [comment]
My only quibble is that the l/1/I degeneracy is the first thing you see in the logo – we're not 
icviz 😬

**Comment by kecnry** [comment]
> My only quibble is that the l/1/I degeneracy is the first thing you see in the logo

hmmm good point... the only way around that might be a script l if something is available in the same font.  @Jenneh any ideas?

**Pull request merged**